SpatialGDK/Source/SpatialGDKFunctionalTests/SpatialGDK/UNR-3761/SpatialTestNetReference/SpatialTestNetReference.cpp

```diff
@@ -212,13 +207,6 @@ void ASpatialTestNetReference::BeginPlay()
 				}
 			}, 5.0f);
 	}
-	
-	AddStep(TEXT("SpatialTestNetReferenceServerCleanup"), FWorkerDefinition::Server(1), nullptr, [this](ASpatialFunctionalTest* NetTest) {
-		// Possess the original pawn, so that the spawned character can get destroyed correctly
-		OriginalPawn.Key->Possess(OriginalPawn.Value);
-
-		FinishStep();
-	});
```

Comment:
Why was it necessary, and why has it been removed?

Reply:
This was originally added because I did not know that the RegisterAutoDestroy function can be used correctly with all Actors. I then realised it is safe to use and therefore not requiring special casing of the Pawn.

Reply:
I think this may still be necessary - this is mostly for re-running purposes, and if having multiple tests in the map, as other tests may expect the default pawn to be possessed.
So if we ever want to move this test to the SpatialNetworkingMap, this may need to be there.

Reply:
Oh yes, you are absolutely right, failed to realise this, thank you for pointing it!